packages/app/src/debug/DebugSpec.vue

```diff
@@ -80,8 +75,8 @@ export interface Spec {
 }
 
 export interface TestResults {
-  id: string
-  titleParts: string[]
+  readonly id: string
```

Comment:
Are these readonly for the purpose of making GraphQL types happy?

I've run into this a lot, I recently found out you can turn off the readonly config, which seems to make things much nicer without any real downsides (just be careful not to mutate props, pretty easy).

Here's how I turned it off in my other project (also using Nexus and GraphQL Codegen) https://github.com/lmiller1990/eleutheria/blob/main/packages/frontend/codegen.yml#L5

Might be good to do this at some point, either here or another PR.

Reply:
Yes, just to make types happy.  I would definitely like to have a conversation to determine if that configuration change would make sense or if we should handle that some other way.